testnets/osmosistestnet5/chain.json

```diff
@@ -0,0 +1,155 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "osmosistestnet",
+  "status": "live",
+  "network_type": "testnet",
+  "pretty_name": "Osmosis Testnet",
+  "chain_id": "osmo-test-5",
+  "bech32_prefix": "osmo",
+  "daemon_name": "osmosisd",
+  "node_home": "$HOME/.osmosisd",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "uosmo",
+        "fixed_min_gas_price": 0,
+        "low_gas_price": 0,
+        "average_gas_price": 0.025,
+        "high_gas_price": 0.04
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "uosmo"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/osmosis-labs/osmosis",
+    "recommended_version": "v15.0.0",
+    "compatible_versions": [
+      "v15.0.0"
+    ],
+    "cosmos_sdk_version": "0.45",
+    "consensus": {
+      "type": "tendermint",
+      "version": "0.34"
+    },
+    "cosmwasm_version": "0.30",
+    "cosmwasm_enabled": true,
+    "genesis": {
+      "genesis_url": "https://github.com/osmosis-labs/networks/raw/main/osmo-test-4/genesis.tar.bz2"
```

Comment:
would have a new genesis, no?

Reply:
sorry yeah i missed that, following this doc here: https://github.com/osmosis-labs/testnets/tree/main/testnets/osmo-test-5

updated to the new genesis, seems they are hosting differently now: https://genesis.osmotest5.osmosis.zone/genesis.json